crates/core/src/state/pair.rs

```diff
@@ -22,7 +22,7 @@ pub struct Pair(Token, Token);
 
 impl Pair {
     pub fn new(token_a: Token, token_b: Token) -> Self {
-        let zero2one = token_a.address < token_b.address;
+        let zero2one = token_a.symbol < token_b.symbol;
```

Comment:
if not doing zero2one then just get rid of this variable and move to the if condition